### ISSUE TITLE
[Bug] Trying to post in a 404'd thread = crash

### ISSUE THREAD

**Issue by floppyD** [Adamantcheese/Kuroba#244]
```
E/Kuroba | ChanThreadLoader: Could not find savedThread for loadable Loadable{id=147, mode=0, board='g', no=72062332, title='/g/ –  ', listViewIndex=3, listViewTop=-411, lastViewed=72062533, lastLoaded=72062571, markedNo=-1, dirty=false}
I/Kuroba | ChanThreadLoader: Loading error
I/Kuroba | ChanThreadLoader: com.android.volley.ServerError
I/Kuroba | ChanThreadLoader: 	at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:160)
I/Kuroba | ChanThreadLoader: 	at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:106)
E/Kuroba | UNCAUGHT: java.lang.NullPointerException: Attempt to read from field 'com.github.adamantcheese.chan.core.model.orm.Loadable com.github.adamantcheese.chan.core.model.ChanThread.loadable' on a null object reference
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.layout.ThreadLayout.showPosts(ThreadLayout.java:269)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.core.presenter.ThreadPresenter.showPosts(ThreadPresenter.java:1117)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.core.presenter.ThreadPresenter.onListScrolledToBottom(ThreadPresenter.java:623)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.layout.ThreadListLayout.lambda$onRecyclerViewScrolled$0$ThreadListLayout(ThreadListLayout.java:164)
E/Kuroba | UNCAUGHT: 	at com.github.adamantcheese.chan.ui.layout.-$$Lambda$ThreadListLayout$wCwyHJXO34_chrEhTvPk0rZTQGs.run(lambda)
E/Kuroba | UNCAUGHT: 	at android.os.Handler.handleCallback(Handler.java:751)
E/Kuroba | UNCAUGHT: 	at android.os.Handler.dispatchMessage(Handler.java:95)
E/Kuroba | UNCAUGHT: 	at android.os.Looper.loop(Looper.java:154)
E/Kuroba | UNCAUGHT: 	at android.app.ActivityThread.main(ActivityThread.java:6186)
E/Kuroba | UNCAUGHT: 	at java.lang.reflect.Method.invoke(Native Method)
E/Kuroba | UNCAUGHT: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:889)
E/Kuroba | UNCAUGHT: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:779)
E/Kuroba | UNCAUGHT: .
E/Kuroba | UNCAUGHT: ----------------------------------------
E/Kuroba | UNCAUGHT: END OF CURRENT RUNTIME MESSAGES
E/Kuroba | UNCAUGHT: ----------------------------------------
E/Kuroba | UNCAUGHT: .
E/Kuroba | UNCAUGHT: Android API Level: 25
E/Kuroba | UNCAUGHT: App Version: v4.5.3-dev Official
E/Kuroba | UNCAUGHT: Phone Model: Motorola MotoG3
E/Kuroba | ChanThreadLoader: Could not find savedThread for loadable Loadable{id=147, mode=0, board='g', no=72062332, title='/g/ –  ', listViewIndex=3, listViewTop=-411, lastViewed=72062533, lastLoaded=72062571, markedNo=-1, dirty=false}
I/Kuroba | ChanThreadLoader: Loading error
I/Kuroba | ChanThreadLoader: com.android.volley.ServerError
I/Kuroba | ChanThreadLoader: 	at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:160)
I/Kuroba | ChanThreadLoader: 	at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:106)
E/Kuroba | ChanThreadLoader: Could not find savedThread for loadable Loadable{id=147, mode=0, board='g', no=72062332, title='/g/ –  ', listViewIndex=3, listViewTop=-411, lastViewed=72062533, lastLoaded=72062571, markedNo=-1, dirty=false}
I/Kuroba | ChanThreadLoader: Loading error
I/Kuroba | ChanThreadLoader: com.android.volley.ServerError
I/Kuroba | ChanThreadLoader: 	at com.android.volley.toolbox.BasicNetwork.performRequest(BasicNetwork.java:160)
I/Kuroba | ChanThreadLoader: 	at com.android.volley.NetworkDispatcher.run(NetworkDispatcher.java:106)
```

**Comment by K1rakishou** [comment]
So you were viewing a thread that got 404ed and then tried to post in it, is this correct?

**Comment by floppyD** [comment]
It got 404'd the moment I was going to post.

**Comment by K1rakishou** [comment]
Should be fixed in #235
